certified-connectors/Advanced Data Operations/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1389 @@
+        "data"
+      ]
+    },
+    "jsonToText": {
+      "type": "object",
+      "properties": {
+        "headerRow": {
+          "type": "boolean",
+          "description": "Contains header row",
+          "title": "Header Row",
+          "enum": [
+            true,
+            false
+          ],
+          "x-ms-visibility": "advanced"
+        },
+        "rowSeparator": {
+          "type": "string",
+          "description": "The character(s) separating each row, default = \\n",
+          "title": "Row Separator",
+          "x-ms-visibility": "advanced"
+        },
+        "filter": {
+          "$ref": "#/definitions/filter"
+        },
+        "sortOrder": {
+          "$ref": "#/definitions/sortOrder"
+        },
+        "schema": {
+          "$ref": "#/definitions/schema"
+        },
+        "advancedOptions": {
+          "$ref": "#/definitions/advancedOptions"
+        },
+        "data": {
+          "type": "string",
+          "description": "Delimited string table",
+          "title": "Data"
+        }
+      },
+      "required": [
+        "data"
+      ]
+    },
+    "jsonToCsv": {
+      "type": "object",
+      "properties": {
+        "headerRow": {
+          "type": "boolean",
+          "description": "Contains header row",
+          "title": "Header Row",
+          "enum": [
+            true,
+            false
+          ],
+          "x-ms-visibility": "advanced"
+        },
+        "rowSeparator": {
+          "type": "string",
+          "description": "The character(s) separating each row, default = \\n",
+          "title": "Row Separator",
+          "x-ms-visibility": "advanced"
+        },
+        "escapeCharacter": {
+          "type": "string",
+          "description": "The escape character for values that contain the field separator value.",
+          "title": "Escape Character",
+          "x-ms-visibility": "advanced"
+        },
+        "filter": {
+          "$ref": "#/definitions/filter"
+        },
+        "sortOrder": {
+          "$ref": "#/definitions/sortOrder"
+        },
+        "schema": {
+          "$ref": "#/definitions/schema"
+        },
+        "advancedOptions": {
+          "$ref": "#/definitions/advancedOptions"
+        },
+        "data": {
+          "type": "string",
+          "description": "Delimited string table",
+          "title": "Data"
+        }
+      },
+      "required": [
+        "data"
+      ]
+    },
+    "parquetToJson": {
+      "type": "object",
+      "properties": {
+        "filter": {
+          "$ref": "#/definitions/filter"
+        },
+        "sortOrder": {
+          "$ref": "#/definitions/sortOrder"
+        },
+        "schema": {
+          "$ref": "#/definitions/schema"
+        },
+        "advancedOptions": {
+          "$ref": "#/definitions/advancedOptions"
+        },
+        "data": {
+          "type": "string",
+          "description": "Base64 encoded string  containing contents of a parquet data file.",
```

Comment:
@brad302 please make sure that descriptions are grammatically correct and are free of typographical errors. There's an extra whitespace in this string. Please refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) for more information.

Reply:
Thanks @v-EgorKozhadei, that's done.